### DISCUSSION TITLE
View - Image Viewer

### DISCUSSION THREAD

**Discussion by ruebyi** [outdoorbits/little-backup-box#182] [Ideas]
Hi!
Thank you for implanting View! I love to rate my pictures on the way!
Right now I'm searching for a way to view also my RAW files - or I need a way to copy the Ratings to my RAWs at home on my Mac from the JPGs to the RAW files.

Any Ideas?
Tobias

**Comment by outdoorbits** [comment]
Hi, what type of raw are your images? If it's not Nikon NEF, could you send me a raw please?

**Comment by ruebyi** [comment]
Thank you!
I‘m Talking about Olympus and Panasonic RAWs - Where should I pur The raws? Here as an attachment?

**Comment by outdoorbits** [comment]
You could try to send it to ***@***.*** If the attachment is too large, please send me an email, I would reply you a link to upload the files.

**Reply by ruebyi** [reply]
Sorry, where? The Mail is not visible

**Comment by outdoorbits** [comment]
It's filtered by GitHub? Ok, you will find my mail in nearly every file in the header of the code.

**Reply by ruebyi** [reply]
Seems like that!
I sent you two Mails with one RAW each.

Tobias

**Comment by outdoorbits** [comment]
It seems, your raws did not contain preview-images. I replaced to old method (extracting the embedded preview-image) by another, converting thumbnails directly from raw. It's a bit slower but of course much more useful. Writing ratings into your raws worked. Does this work for you?

**Comment by ruebyi** [comment]
that sounds awesome! I will test it!

**Comment by ruebyi** [comment]
It's working great and an absolute Gamechanger for me!

**Comment by robertonhub** [comment]
Hello,
I just set up a little backup box (still waiting for the front cover), and it seems to be working great. Thank you very much for this fantastic helper!
You might already have received a lot of requests for more RAW-support, but maybe... While my Sony ARWs are thumbnailed alright, the Canon CR3s are left black. Do you think, you can change this? Of course, I can send an example file, but that will be quite big... (about 30MB).
Best regards
Robert

Raspi 4 B 2GB
Target: WD My Passport SSD 2TB (WDBAGF0020BSL)
Sources: SDXC II Sony Tough M 128GB, SDHC I SanDisk Ultra 32GB 
I did not yet get the iPhone backup running...

**Comment by outdoorbits** [comment]
dcraw has a known issue converting cr3. But it's known since years. There is another tool, LibRaw,  but I have to read up about licence conditions, looks a bit difficult ...

**Comment by outdoorbits** [comment]
O.k., I just replaced dcraw by libraw. My tests against Nikon NEF have been successful. Could everybody please test other RAWs? ... and give feedback of course, thank you!

**Reply by ruebyi** [reply]
WOW you are fast!
ORFs from Olympus are working fine.
It feels like libraw is a bit slower, than dcraw, but most important: it works!

**Comment by outdoorbits** [comment]
There was also an issue about canon CR3 files. Huge exif data killed the formatting of single view. It's fixed also.
Thank you all for your feedback and your bug reports, they help a lot!

**Reply by robertonhub** [reply]
And thank you for the fast implementation and bug fixing!
